eng/pipelines/official.yml

```diff
@@ -31,6 +31,7 @@ parameters:
 extends:
   template: templates/pipeline.yml
   parameters:
+    isOfficialBuild: true
```

Comment:
Please correct me if I am wrong, official build pipeline has a variable named `IsOfficialBuild` set to `true`. Can we use that variable instead of defining a new parameter.

Reply:
Unfortunately no, that won't work. For the same reason that I had to [duplicate all the test jobs "Run Tests" steps with "continue on error" and "stop on error" versions](https://github.com/NuGet/NuGet.Client/blob/0d161cc61054fcc3739d629ce6930263a573b3ae/eng/pipelines/templates/Tests_On_Linux.yml#L2-L18), we have to do the same here. AzDO simply does not support reading runtime variables for `continueOnError`, it must be a "compile time" parameter.

In a future PR, I think I can replace these duplicated "Run tests" steps with a single one where `continueOnError: true` is replaced with `continueOnError: ${{ parameter.isOfficialBuild }}`.

parameters and variables are not interchangable 😞